src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.3'
+ModuleVersion = '0.1.4'
```

Comment:
no need to upgrade module version, will be updated during release

Reply:
Ok, reverted.